src/server/api/controllers/user.controller.js

```diff
@@ -11,9 +11,7 @@ const getUsers = async req => {
   try {
     return await knex("users")
       .select("users.email", "users.name", "user_roles.role")
-      .join("user_roles", {
-        "users.role_id": "user_roles.id"
-      });
+      .join("user_roles", { "users.role_id": "user_roles.id" });
```

Comment:
Do you have installed Prettier in the vs code? Because I beilive most of us have and you kinda changing the formats here. Which we might have problem in the future, because Prettier it does format automatically.

Reply:
We are here and discussing it. and thanks for mentioning.

Reply:
Please review the code and if it was something wrong let me know. ;)